dotCMS/src/main/java/com/dotcms/publisher/business/PublisherQueueJob.java

```diff
@@ -292,10 +303,19 @@ private void updateAuditStatus(final List<Map<String, Object>> bundlesInQueue) t
 	@NotNull
 	private void updateBundleStatus(final PublishAuditStatus bundleAudit,
 									final Map<String, Map<String, EndpointDetail>> endpointTrackingMap,
-									final GroupPushStats groupPushStats, final List<Map<String, Object>> bundlesInQueue) throws DotDataException, DotPublisherException {
+									final GroupPushStats groupPushStats, final List<Map<String, Object>> bundlesInQueue)
+			throws DotDataException, DotPublisherException, DotSecurityException, LanguageException {
 		Status bundleStatus;
 		final PublishAuditHistory localHistory = bundleAudit.getStatusPojo();
 		final String auditedBundleId = bundleAudit.getBundleId();
+		//Info need to generate Growl Notification
+		final Bundle bundle = APILocator.getBundleAPI().getBundleById(auditedBundleId);
+		final boolean isBundleNameGenerated = bundle.getName().startsWith("bundle-");
+		String notificationMessage = "";
+		String notificationMessageArgument = "";
+		long notificationLife = 5000;
+		MessageSeverity notificationSeverity = MessageSeverity.SUCCESS;
+
```

Comment:
All these arguments could become an object called `NotificationParams` and use a builder pattern. 
E.g. 
```new NotificationParamsBuilder().withMessage("").withMessageArgument("").withNotificationLife("").withSeverity(SUCCESS)```

and then call the .build() when passing it to the private method `sendNotification(builder.build())`

Reply:
using builder